### DISCUSSION TITLE
For anyone looking to how to move beatmaps and skins from PC to phone, here's how

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [ppy/osu#28123] [Ideas]
1. Navigate to `C:\Users\YOU\AppData\Roaming\osu`
2. Connect your phone to PC and naviage to `Android\data\sh.ppy.osulazer\files`
3. On the phone storage delete the `files` folder (not the one you're in, the `sh.ppy.osulazer\files\files` one
    If you're not sure if it's the right one, It contains bunch of differently numbered folders like 0-9 and a-z
4. Delete `client.realm` and `client.realm.lock`
5. Now copy `files`, `client.realm` and `client.realm.lock` from PC's osu folder to the phone essentially replacing what we deleted
6. Now you should have all skins and beatmaps transfered

**IMPORTANT NOTE:** `client.realm` is an odd thing and if you don't delete `files` folder and just merge the folders, it will get very confused about incorrectly placed files

**Comment by FirecatStudios** [comment]
This in fact also works in reverse from phone to PC just in the opposite order instead of moving files from PC to phone just tried it got my phone data for maps and such onto my PC version

**Reply by FirecatStudios** [reply]
Side note you might lose some audio and beat maps in the process as I just experienced.
